web/modules/custom/portland/portland.module

```diff
@@ -213,7 +225,7 @@ function portland_group_presave(GroupInterface $group) {
     }
     // Check if this is a new group or an existing group whose shortname has been modified
     if( $originalShortname !== $newShortname) {
-        $cleanShortname = \Drupal::service("pathauto.alias_cleaner")->cleanString($newShortname);
+        $cleanShortname = $newShortname;
```

Comment:
I wonder if we want to take this chance to rename this field to group_path. I'm a little worried that all of our references to shortname don't really make sense in this new context. 🤔